generate_quantitative_analysis_map.py

```diff
@@ -38,3 +40,17 @@
     with open(individuals_json_input_path) as f:
         individuals = TracedDataJsonIO.import_jsonl_to_traced_data_iterable(f)
     log.info(f"Loaded {len(individuals)} individuals")
+
+    individuals_quantitative_map = {}
+    messages_quantitative_map = {}
+
+    individuals_quantitative_map['INDIVIDUAL_SHA'] = SHAUtils.sha_string(f'{individuals}')
```

Comment:
This won't match computing the sha from the command-line. Please could you share the characters in the file rather than an interpretation of it.

Reply:
I'm not sure that 'INDIVIDUAL_SHA' is a very explanatory name. I think something like 'Origin_Individual_tracedata_jsonl_sha' would be better